### DISCUSSION TITLE
coc colors for the floating info windows on startup

### DISCUSSION THREAD

**Discussion by kenielf** [neoclide/coc.nvim#4002] [Q&A] [answered]
HI, I was wondering if there is any way to change the colors of the popups that appear when first initializing coc inside a file, like below:
![image](https://user-images.githubusercontent.com/61657416/182773187-7ed1e70a-fef2-4ad5-a5f5-1428fc650094.png)
the `:h` command for it should suffice, thanks!

**Comment by chemzqm** [comment, marked as answer]
https://github.com/neoclide/coc.nvim/blob/3dde9b8ccab9acee0ef085087b26fa98401eefeb/doc/coc.txt#L3493

**Reply by kenielf** [reply]
Thank you! This is how it looks now
![image](https://user-images.githubusercontent.com/61657416/182871630-82eef7c3-e01d-48f3-ad42-0e0c8af99482.png)
```vim
highlight CocNotificationProgress guibg=#4c566a guifg=#5e81ac
highlight CocNotificationButton guibg=#4c566a guifg=#5e81ac
highlight CocNotificationError guibg=#4c566a guifg=#5e81ac
highlight CocNotificationWarning guibg=#4c566a guifg=#5e81ac
highlight CocNotificationInfo guibg=#4c566a guifg=#5e81ac
```
A follow-up question, what keyword do I use to change the background color of the window?

**Reply by chemzqm** [reply]
`:h coc-config-notification-highlightGroup`
